### ISSUE TITLE
Enable and fix a11y e2e tests

### ISSUE THREAD

**Issue by raftmsohani** [raft-tech/TANF-app#5407]
## Background ##
Follow up to #5308, there are a11y errors that need to be resolved before enabling a11y accessibility module. 

### Why It Matters ###
Cypress tests for accessibility errors

### What's Changing ###
Cypress will perform accessibility tests according to a11y requirements.

### Acceptance Criteria ###
_Create a list of functional outcomes that must be achieved to complete this issue_
- [ ] _Outcome 1_
- [ ] _Outcome 2_
- [ ] _Outcome 3_
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary
- [ ] [Release digest](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/JSIbkxujKUxhKv2cXFEg/) has been updated with testing steps & relevant media

### Usability Testing Criteria ###
_PM/UX/OFA Only: Create a list of expected user behaviors that should be confirmed when UX and/or PM is testing this ticket_
- [ ] _Outcome 1_
- [ ] _Outcome 2_
- [ ] _Outcome 3_

### Notes ###
_Add additional useful information, such as related issues and functionality that isn't covered by this specific issue, and other considerations that will be helpful for anyone reading this_ 
- _Note 1_
- _Note 2_
- _Note 3_

### Supporting Documentation ###
_Please include any relevant log snippets/files/screen shots. This should also include files needed to adequately test the successful completion of this ticket._
- _Doc 1_
- _Doc 2_

### Open Questions ###
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_ 
- _Open Question 1_
- _Open Question 2_
